nix: upgrade nixpkgs

jakubgs opened the pull request:
Significant updates:

- Git: `2.25.1` to `2.28.0`
- Make: `4.2.1` to `4.3`
- Glibc: `2.30` to `2.31`
- PatchElf: `0.9` to `0.11`
- GoLang: `1.14.0` to `1.14.7`
- NodeJS: `v12.16.2` to `v12.18.3`
- Clojure: `1.10.1.507` to `1.10.1.645`
- OpenJDK: `8u242-b08` to `8u265-ga`
- GoMobile: `20200329` to `20200622`

This is done in advance of Android SDK/NDK updates when https://github.com/NixOS/nixpkgs/pull/89775 is merged.

Some local testing would be much appreciated.

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | ead39ee8 | [#1](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-11119/1/) | 2020-08-27 13:46:09 | ~11 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-11119/1/consoleText) |

jakubgs commented:
Looks like we have two problems. The big one is with `gomobile` not wanting to build for `ios/arm` for some reason
```
[2020-08-27T13:43:36.889Z] Building for targets: ios/arm,ios/arm64,ios/amd64
[2020-08-27T13:43:36.889Z] 
[2020-08-27T13:43:37.598Z] /nix/store/w0wisvhmcdvxjllg2c3qzgwcc6xqqsv7-gomobile-20200329-4c31acb/bin/gomobile: 
    go [-e -json -compiled=true -test=false -export=false -deps=false -find=true -tags=ios -- github.com/status-im/status-go/mobile]: 
    exit status 2: cmd/go: unsupported GOOS/GOARCH pair darwin/arm
```
The other one is with failure to download `clj-kondo` within Nix shell:
```
[2020-08-27T13:43:39.276Z] INFO: Running clj-kondo for the first time; downloading the actual binary
[2020-08-27T13:45:02.367Z] ERR Error communicating with URL https://github.com/borkdude/clj-kondo/releases/download/v2020.01.13/clj-kondo-2020.01.13-macos-amd64.zip Error: connect ETIMEDOUT 140.82.121.4:443
[2020-08-27T13:45:02.368Z] error Command failed with exit code 1.
```

jakubgs commented:
I tried upgrading `gomobile` from `20200329` to `20200801` but that didn't work:
```
[2020-08-27T14:11:10.393Z] Building for targets: ios/arm,ios/arm64,ios/amd64
[2020-08-27T14:11:10.393Z] 
[2020-08-27T14:11:10.727Z] /nix/store/1v8g94rfid7pngz849j5r3ia7d2hqzdi-gomobile-20200801-973feb4/bin/gomobile:
    invalid -target="ios/arm,ios/arm64,ios/amd64": unsupported arch: "arm"
```
But the `clj-kondo` issues seems to have resolved itself.

jakubgs committed:
nix: upgrade nixpkgs

Significant updates:
- Git: `2.25.1` to `2.28.0`
- Make: `4.2.1` to `4.3`
- Glibc: `2.30` to `2.31`
- PatchElf: `0.9` to `0.11`
- GoLang: `1.14.0` to `1.14.7`
- NodeJS: `v12.16.2` to `v12.18.3`
- Clojure: `1.10.1.507` to `1.10.1.645`
- OpenJDK: `8u242-b08` to `8u265-ga`
- GoMobile: `20200329` to `20200622`

Signed-off-by: Jakub Sokołowski <jakub@status.im>

jakubgs commented:
According to [this issue](https://github.com/golang/go/issues/39575) and Go `1.15` accouncement:

>As announced in the Go 1.14 release notes, Go 1.15 drops support for 32-bit binaries on macOS, iOS, iPadOS, watchOS, and tvOS (the darwin/386 and darwin/arm ports). Go continues to support the 64-bit darwin/amd64 and darwin/arm64 ports.
https://tip.golang.org/doc/go1.15#darwin

Which might be a serious pain for us. Which means we have to lock Go at `1.14`.

jakubgs commented:
It appears that the https://github.com/golang/mobile/commit/588b2b2f commit in `gomobile` breaks `ios/arm` builds regardless of Go verison.

So we won't be able to upgrade beyond https://github.com/golang/mobile/commit/33b80540 without losing ARM 32bit support for iOS.

flexsurfer reviewed (approved):
tested locally works fine

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 98
Failed tests: 0
Passed tests: 98
```
<h3>Passed tests (98)</h3><details><summary>Click to expand</summary><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630329">test_decline_transactions_in_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c1e27c0aa08444cda73a6da2ccac6c13?auth=807506be49fb15145b420d101a937d05">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/87eab70e585a4c6da84924caaf91e9b0?auth=2ae5b989433ce6ec74a78b399779f458">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630277">test_delete_chats_via_delete_button</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/684c16887821433592244a7047738b6b?auth=27f03860d38882afa4d654193b695e8a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630237">test_can_see_all_transactions_in_history</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/97f84f0b64f342f8b231eddeff7fe68f?auth=22b7680a1863fa5b8c613297bfeb1796">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630305">test_install_pack_and_send_sticker</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5c479d235fba4778b9b892160ba3cc97?auth=504372d79cd856cadf711d5d07b7e53c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/763cb879f2d94a5a880ba750b46b622e?auth=6bddd09c98fcdae3e7af3939e85252a3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630235">test_login_with_new_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6fa6d02f86b2493c9ae791d2e8d595da?auth=f12504a9f5b4fdac1562f17decfeae85">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630275">test_timestamp_in_chats</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bcc4c64865164624b68da1012fda8075?auth=704e5e367a3ef8f284e3bd55e2b023ee">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/85279f62ae4345e2a44d29a4b085c37c?auth=1e3c0e3e0a05da6ce9cb4f24a675e583">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630289">test_password_in_logcat_sign_in</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6063b482931d4a45a5d83f1680006fa7?auth=dc6ddb933dc04424c8bc272113fd2f60">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630267">test_logcat_recovering_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/360d3738b65644dca873470f9868451f?auth=701ca54751e43861b96623e42cd76c31">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630276">test_public_chat_clear_history</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cc4562ae7123434eac966f2894dfd9a2?auth=d51807550f9b975160e7b1c8b6de392b">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/c391fd319cb94cebbf4f7d78cb770d2b?auth=e8b9642bba4d5ed4bd76375292cc592f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630279">test_need_help_section</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f8f2853265684382a81368635b09e388?auth=d83d248c4818bbed20a10c6e0354efe4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630269">test_send_and_open_links</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fa76d4de2f2d470ca65e76b848a75f0b?auth=ed68380505c8b6999f033f85e78f3c7e">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/3d58b935b64a441d8e32db3aa0a2db77?auth=b8f91cbbac43e41d502485e9a6cacee3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630318">test_mobile_data_usage_popup_stop_syncing</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fde3061969a542a3bb4897e503303bc3?auth=d63a02b2eccfc30526672588eecec533">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>13. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630342">test_keycard_send_eth_in_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/22faa76023494942a4f0e9adf34217a0?auth=fc83f91afa5f7b37f45b275e579b7dae">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/c76fe798b46e419f965c7f6e541080d2?auth=1299727d0b03dc7908569f2421142b1b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>14. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630319">test_mobile_data_usage_settings</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/dac341822c724f5a8b09ea4e9e6ee311?auth=5e510dd7b9593da083a36554203f582f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>15. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630226">test_create_new_group_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/68cab345dab446aa935f972163a087e1?auth=a5ac3134615c3c45dc5378decf800072">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8ab5392d0f07475ca13d5812ac5305db?auth=7d96dc5f08b7d9801165a3539c4acbef">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>16. test_keycard_send_eth_to_ens</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b68632cea4a74fc69cd3ae5d2791c149?auth=11802c1fbccfc17593c3a85c1dc15cb0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>17. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630228">test_user_can_switch_network</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f06bf2ee18614ae1be61ee5da87dc1c6?auth=b1c028482fedeedf5d1c5adc97c9f13e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>18. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630299">test_mobile_data_usage_popup_continue_syncing</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d85e03fab5e94e1ea11549421510f987?auth=9e8579a2cc9c7f55cea75f2ce33be451">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>19. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630271">test_home_view</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b9e2365514824da9a5e23c4abcf1519e?auth=165e587618b468c8c426a18b91fa7478">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>20. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630242">test_open_google_com_via_open_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d9285fb073b34d8eb3c5e398bd2b62c1?auth=163b0374cac5b5095c36744edcaae35a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>21. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630244">test_share_contact_code_and_wallet_address</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ea6f567e0536408396fed87b3cac29fe?auth=2042318bd714a13111c0e807bd6d2ca5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>22. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630283">test_connection_is_secure</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/170420fe4156430b920521acb57b2a07?auth=f841116619a681d79f318f84fee90787">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>23. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630239">test_add_to_contacts</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/13113962c3a6423cbfd9e8094f9183f9?auth=d3226eaaff94312d4981bdf8aa085503">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/03ad6e30165c4e308564c76ee9b0cd22?auth=d7cea55d3bb68e13a478b97e0d485f66">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>24. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630331">test_add_account_to_multiaccount_instance_private_key</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1d6603549eb24823a228a4f3feab574a?auth=7a585a571405ff04ffff1d092c9574c9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>25. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630259">test_refresh_button_browsing_app_webview</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6498b707a38e408584719cfb210352ae?auth=2e1b4541ed7d00b7eb66373297f9f5e5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>26. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630335">test_push_notification_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1cba38c2fffe42e999b4c07de651bd87?auth=b777715c3a38d773464f8fc2a92db406">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/3c4da9bc06eb427ca2bddf8e077d9276?auth=b1a533fc70780e33a5825d43c31d3f80">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>27. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630338">test_keycard_send_two_transactions_one_after_another_in_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bd105825ca6f48aea23d48ca7f4b5e02?auth=9aa0a98f633d7c7ea43615d652774e85">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>28. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630292">test_can_add_existing_ens</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3593b53232264c02909e23ae185e37d7?auth=7cd92c60d179da5cd8151615c70bdbeb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>29. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630286">test_transaction_wrong_password_wallet</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/55be2777f6de40c39feaf62ddb9116b2?auth=90923a083f28aa7eca63f0d2dc05bb41">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>30. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630236">test_public_chat_messaging</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b849c0bcf779452f823884d1e2b4fffd?auth=0b8488528bf665fec27bebfca9ab700b">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/9d6683a4311547d384c968c97d184c6f?auth=44f0179640dec1e46b72a27968d80c00">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>31. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630262">test_unread_messages_counter_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/557a3633ed614f58bec4581d658aab00?auth=04fa9942f823390fbcc88a1c041c9a6f">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/1e5808f20eb44e14868083610cc5112f?auth=499913286c7473a623e45743a647c200">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>32. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630339">test_keycard_send_eth_from_wallet_to_address</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ea2d846dbad849bdaff0fac3a20e7ac0?auth=4b2a7fcea4240476f22bf63025bdbb76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>33. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630340">test_keycard_fetching_balance_after_offline</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/aa8850f1d98442ae899127595e1cc58e?auth=50873e196bead329c120530b66ef964e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>34. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630323">test_keycard_send_transaction_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2634041fdfd340a189899f937dbe6431?auth=226e9d99178649a5c2631d52e2adb45c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>35. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630231">test_text_message_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6039de3ad6df4e80a3623494a13e6c79?auth=f7f5ac4c69578697cafe0e2b260d6c30">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/2accf77898ac47efb476eb850737f4b8?auth=eb23185f5c493f118379381a8dc501bb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>36. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630306">test_can_use_purchased_stickers_on_recovered_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7c01629eed90408794a5ecff8232e5bf?auth=f68c6064363e0deb68bc22f4a1d72fd6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>37. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630274">test_open_transaction_on_etherscan</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5a0af4549efc40ff8de4e07c4649ff31?auth=91cdafd20498700a1cc206c19a7cfd93">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>38. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630295">test_onboarding_screen_when_requesting_tokens_for_recovered_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/868982a4ed304dd698cf8dc8548f447c?auth=8e2396a9ed8633e948a58b51bb3fa3ac">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>39. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630273">test_contact_profile_view</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1a2a60884a6b42f397eca99573a71556?auth=ea1e2889374a44b2d69a36053dc2bc7b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>40. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630227">test_offline_add_new_group_chat_member</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2ce86529dc47461781ba0f52eceb7f0d?auth=c7b96a33ed856de5d83381de2175bd15">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/693a23da71434ad6bc34be072b0fd340?auth=857e0712177fec29884fa544e56291db">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://saucelabs.com/jobs/7e86270f998441d09040ed8d2a41a509?auth=d18fcca0ab974e81f96295ac66b78bec">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>41. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630290">test_logcat_backup_recovery_phrase</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/91fd6b544da54e8e8b82ab8e8dd41a51?auth=8174b6bfdc2fd8ca86078c5f52645088">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>42. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630302">test_can_recover_keycard_account_card_pairing</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b9c99ec388494981bad2b93866dffa3f?auth=1bb53dafa5c01b33519bb1353afb7d5c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>43. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630284">test_start_chat_with_ens</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ab4ce3591ebd4c71997c9ccdec7b4518?auth=be5c7485a18fea405b4b151fdeae4be5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>44. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630248">test_add_and_remove_contact_from_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7e6e4abc79fe4175b74265c5565b592b?auth=6ed1542084b691ce0419d8258fcb1797">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/5f875e65e80e4525a14f315644264d1a?auth=d7b5455ace4be129eea67127f556de1a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>45. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630330">test_add_account_to_multiaccount_instance_seed_phrase</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/73638489f7e8418e9e5e65f077b284ce?auth=5397bfda2b39187c334af9f5bc63b03a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>46. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630311">test_open_blocked_site</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/65b94dcd75884fa9bc87a7d2f6d62e29?auth=9f751fa8f8d708bc431f67da072ecf47">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>47. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630230">test_open_chat_by_pasting_public_key</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e4ce836661fe4e0b95712cd5756bc381?auth=3218d1cbb2d1e0b0c0b35464a9632b1d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>48. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630304">test_pair_devices_sync_one_to_one_contacts_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d69ee2ad7e234bfbb2e60c02155a0e66?auth=6547ddc8815929ff935b39a51e99ecb4">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/281710598c3a46ddae553d9e9cb23249?auth=be7dd51e31348864d14a6692aff417d1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>49. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630325">test_keycard_sign_message_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/47799e21291644d9bd17a8480cf5baf6?auth=a8200ef0f9080296bce3a1dfd7d8a6b2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>50. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630238">test_send_non_english_message_to_newly_added_contact</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5e9ded534d0b44ff91bc9770b1a67b22?auth=2c679da3b4077ea2f782e0102fb5096b">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/f07591f8c3bf44b78b9ba70e30515dcc?auth=fc8adaa9a2ee54fc62261450f10d90ce">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>51. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630285">test_ens_username_recipient</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/71861046a9574b838b82d18eb421efd6?auth=f2c3d54711d4e92fa6063fab420b5dda">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>52. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630308">test_fetch_more_history_in_empty_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c5cd3c1f1ed042408903abf846f7ebdb?auth=d28cf82dfc283256851cc1cb403048a6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>53. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630281">test_open_public_chat_using_deep_link</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1696abd4a31a4a208eca653b12a4adcc?auth=dab85ef8aeee813b9d502326fe78b502">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>54. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630260">test_switch_users_and_add_new_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6f95b07038044706b7a536ce3b57b58b?auth=0b0d3486120bbd341348e0bb25957548">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>55. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630315">test_add_account_to_multiaccount_instance_generate_new</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/aababd8fce6a4424a6a71c3b1e6c5a5e?auth=50899fa60e79d4f6f757993ad98af4fc">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>56. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630270">test_copy_contact_code_and_wallet_address</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1f2610f5500a4bc5b1c36ea2fbb6de1d?auth=1b5e2820841ee9c1d6ee022aa0eb0e0c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>57. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630233">test_send_transaction_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6846b55174a743698305eb67ccd90380?auth=cdb7243bed7eb44af4b7fe2827600d3c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>58. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630301">test_send_two_transactions_one_after_another_in_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c11b5250a5bf4c159f9d848fd299590c?auth=cbdad174e2845d0a424e84d49b3f7719">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>59. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630344">test_image_in_one_to_one_send_save_reply</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8f44cde422f843099c40efa68cfc146d?auth=00511dd69ce8d029944f22b6cb39174f">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/4bb80b7c323d4463846955eefac5aaa9?auth=84e64594acf61bdb8cf413b3e18bba17">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>60. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630296">test_add_new_keycard_account_and_login</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c7fcf199ac90408a87f769fb6270e05a?auth=927943a6427d5991acab663901f85270">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>61. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630278">test_long_press_delete_clear_all_dapps</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/feef90e3e9394ebfb3e931cb779320e9?auth=99465c4e1405a4992a916b38bf5a28a2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>62. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630240">test_copy_and_paste_messages</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/585f4656a92f45cca644932a53fde4d8?auth=cb0068c000df63f36044962dde70338e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>63. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630243">test_back_forward_buttons_browsing_website</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9f30f49d4c414964abe833df2a77b63a?auth=47a0c8e9ef6452aaf62b11791f9967f0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>64. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630288">test_password_in_logcat_creating_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1f4352e9e2bd4da684f4c434e1f51276?auth=0c653987a7cab8e42158d65fb1a3336f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>65. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630255">test_sign_message_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d168ca7120ae4fb1a3e054e27b6351a5?auth=6f06fe3e84ca202b411fea56f50e5a8b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>66. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630298">test_dapps_permissions</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/30c0160b2ebe48aa9a13ea3135e09ce4?auth=8726be1476f869a70321aa312605c0a8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>67. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630282">test_request_public_key_status_test_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8913c51cd4e247a7b5f21a32605ed2e3?auth=e0d9fb2324def69b4846f860cb0aac61">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>68. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630241">test_long_press_to_delete_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8cda77f0be1e4d0597282d208f7bf7fb?auth=3cb26f1d09cc3f4faa3c2fda725dc295">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>69. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630268">test_log_level_and_fleet</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/da7b6cffe67d43729711e49dfbf891f5?auth=276adbc182dd95521758ec7e789a2964">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>70. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630345">test_reactions_to_message_in_chats</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d8b0aab28269469a926f55c0d9bd7ac9?auth=b7a825b333c2f569a1e9ab9d1ed77c05">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/b1c135ce12d84cfcb2b15fd48b0ee14b?auth=9acc03f6402328ea7b5d02156923e90c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>71. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630322">test_add_and_delete_watch_only_account_to_multiaccount_instance</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/eae2c4e8dcc34a7f9a4b0dd2901d5be4?auth=25584d213b73082b268371f30518250c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>72. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630343">test_recover_account_from_new_user_seedphrase</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/62abf19814fb47b7bfeb5929d9a5e37f?auth=b295e993827c71fe692b8fccbeccbec5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>73. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630309">test_send_transaction_with_custom_token</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8e3b5370b6bb47d3933131094c2e4384?auth=5cc4a29318e728e2cc8146f5ad1c487d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>74. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630326">test_send_eth_in_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/471597d4cbcf4f8db14b7d66b271bcf4?auth=c896554687f349cc87e64cc1de0b5e2b">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/80b2ac986fce4140b6d6feea7fbd7d94?auth=2d5d2911af9a2f626042b9e9631c08f0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>75. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630300">test_send_two_transactions_in_batch_in_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/63b28ed4b4854350ae141071651db5e5?auth=542fdbb333716509c40891f365930fd2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>76. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630316">test_ens_in_public_and_1_1_chats</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c2b5b9a929354bb8a1aaf5eec771938a?auth=58e7bd56a0ddeadd823292032ad51109">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/d48b85983f834964a54196672703cabe?auth=e665f3ef7830e41b4f58aac77499c8a4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>77. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630307">test_block_user_from_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/35c114ddbc504405b3a990e13c62a999?auth=27f9e28061e34541037c7b3401314b52">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/03ba8f81cd9e43bc9364f8ed7d951e85?auth=42aaf947287bf01b6a9ea2e57f572501">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>78. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630320">test_fetching_balance_after_offline</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cde8e7d451f744209b471e07ade466ed?auth=d858bec2e5e877d77ab6780a1b8929ba">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>79. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630245">test_offline_status</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0f24c57a828e44148cdedd2d88dcf13a?auth=f41cd327a9a2574618f5cab4bc533240">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>80. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630249">test_wallet_set_up</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/144f0b8988854436872b89349468f8c7?auth=8fcd72330a1944ded92fac430a8fd065">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>81. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630254">test_manage_assets</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/584782048c444bb69c5a011080f8e63b?auth=b87253ba135eed93cb052dfb1d306c64">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>82. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630234">test_offline_messaging_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2e666c64d5324f59b8d97a46eb5b7ade?auth=4762324c1248445ecab47cdb5bde479a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/f720a1b594e144a3bcbe150009e62adb?auth=efd8e9674d21c070b827952f8e04f5fa">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>83. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630280">test_account_recovery_with_uppercase_recovery_phrase</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/734807a5767944d0ac27b599b3a746f0?auth=d92b45b87e708bdad634af277e5f1d65">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>84. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630257">test_send_token_with_7_decimals</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0c6671529fab4872bb57016583fa4828?auth=9d29116b59ab7ae03f6462ccfb9ec653">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>85. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630272">test_user_can_see_all_own_assets_after_account_recovering</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1bbd6d8e40dc492bb68a05d1bb397354?auth=36b668ff1d3eee4d6ef1a229b94365bc">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>86. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630264">test_pass_phrase_validation</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1ab1011becca49d094377b9bb4f02c8a?auth=ba02258b0ebeaedba0c1717639592cb7">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>87. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630341">test_keycard_can_see_all_transactions_in_history</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/940aa2d5a5cd461ea1456f70055d535b?auth=91d09d91ff37f5410e964d67c8553f56">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>88. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630337">test_keycard_send_two_transactions_in_batch_in_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fea11b38d7444f84ad3e96ab3fe78095?auth=5d20c485d9729b9b06f9c2e8af5ffde4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>89. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630246">test_send_emoji</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/67d218e8ec2d498fbe5c2f85fd5b2733?auth=98c712cc46dc6c538bc27fd74c5405d7">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>90. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630258">test_filters_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d3211880965e4d3d9409b1eb791a0334?auth=6b7134db70d33cffbfe4bcb220653263">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>91. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630294">test_redirect_to_public_chat_tapping_tag_message</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/16b56ebb77f6475aad2deb8b63204953?auth=9d40c4613dcc35689691f2a79f02fada">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>92. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630321">test_restore_account_from_mnemonic_to_keycard</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/37126f4f68324a71b9e1c7d69e3af545?auth=5ee27170e4a33082b85ad06809d63074">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>93. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630333">test_send_eth_to_ens_in_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cc9630b3889d427caa79d8610460171a?auth=32f767a180bece424647ed08b1f4e720">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>94. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630232">test_send_eth_from_wallet_to_address</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e8e5ab0dc4f4472c9fdd1e829b57910e?auth=8b255bd870a008b50a185c548af6ab94">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>95. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630328">test_request_and_receive_stt_in_1_1_chat_offline</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/976aa2b825f94e3a9b348fb7af619fc5?auth=475f80e1f334b04cdc2f3a1ce910517f">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/a08c8381ebe644cbab0bcacc9813c85b?auth=1fb520d291144111e8e4273eb0663d4c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>96. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630251">test_messaging_in_different_networks</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5af320856b6e4388b8e9241751ff0eca?auth=60e3c01ce59ae80c05875eab32363e19">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/54836984e6dd45249e9cf4924075d8b2?auth=abb6682300a3d2c9e63a537a25f43241">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>97. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630256">test_collectible_from_wallet_opens_in_browser_view</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e546fdccb3544bb59196075d2a61ee72?auth=47117bf7ff5db01eff91e79d4a708f63">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>98. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1630287">test_insufficient_funds_wallet_positive_balance</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/36824dcc7f6847d5b87f1b142b944d68?auth=1dfaa4e6d16daea11229ec2057fb408f">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>
